shanoir-ng-front/src/app/studies/study/study-tree.component.html

```diff
@@ -13,6 +13,7 @@
 -->
 <div class="commands">
     <button class="right-icon dl-button" type="button" (click)="downloadSelected()" [disabled]="!(selectedDatasetNodes?.length > 0)">Download selection<i class="fas fa-download"></i></button>
+    <button class="right-icon dl-button" type="button" (click)="openInViewer()" [disabled]="!(selectedExaminationNodes?.length > 0)" [title]="'Select examinations to open in viewer'">Open in viewer<i class="fas fa-eye"></i></button>
```

Comment:
I propose, if okay: 'Select examinations to compare DICOM files in viewer' and 'View DICOM' and see my other comment, use it everywhere please.